### PR TITLE
Add missing periods

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: docs/core/porting/windows-compat-pack.md [hunk 1]
```diff
@@ -13,12 +13,7 @@ found in the .NET Framework. The *Windows Compatibility Pack* provides many
 of these technologies, so it's much easier to build .NET Core applications and .NET
 Standard libraries.
 
-This package is a logical [extension of .NET Standard 2.0](../whats-new/dotnet-core-2-0.md#api-changes-and-library-support)
-that significantly increases API set and existing code compiles with almost no
-modifications. But in order to keep the promise of .NET Standard ("it is the set
-of APIs that all .NET implementations provide"), this didn't include
-technologies that can't work across all platforms, such as registry, Windows
-Management Instrumentation (WMI), or reflection emit APIs.
+This package is a logical [extension of .NET Standard 2.0](../whats-new/dotnet-core-2-0.md#api-changes-and-library-support) that significantly increases API set and existing code compiles with almost no modifications. But in order to keep the promise of .NET Standard ("it is the set of APIs that all .NET implementations provide"), this didn't include technologies that can't work across all platforms, such as registry, Windows Management Instrumentation (WMI), or reflection emit APIs.
```
Comment (mairaw) [inline docs/core/porting/windows-compat-pack.md]: Was this about the paragraph formatting or a missing period? If the former, should you do the same to the rest of the article?

File: docs/csharp/tutorials/console-teleprompter.md [hunk 2]
```diff
@@ -25,8 +25,7 @@ There are a lot of features in this tutorial. Let’s build them one by one.
 ## Prerequisites
 
 You’ll need to set up your machine to run .NET Core. You can find the
-installation instructions on the [.NET Core Downloads](https://dotnet.microsoft.com/download)
-page. You can run this
+installation instructions on the [.NET Core Downloads](https://dotnet.microsoft.com/download) page. You can run this
```
Comment (mairaw) [inline docs/csharp/tutorials/console-teleprompter.md]: should you reformat the rest of the paragraph too?

File: docs/csharp/tutorials/console-webapiclient.md [hunk 3]
```diff
@@ -30,8 +30,7 @@ If you prefer to follow along with the [final sample](https://github.com/dotnet/
 ## Prerequisites
 
 You’ll need to set up your machine to run .NET core. You can find the
-installation instructions on the [.NET Core Downloads](https://dotnet.microsoft.com/download)
-page. You can run this
+installation instructions on the [.NET Core Downloads](https://dotnet.microsoft.com/download) page. You can run this
```
Comment (mairaw) [inline docs/csharp/tutorials/console-webapiclient.md]: same here

File: docs/standard/glossary.md [hunk 4]
```diff
@@ -60,7 +60,7 @@ A set of libraries that comprise the System.* (and to a limited extent  Microsof
```
Comment (mairaw) [inline docs/standard/glossary.md]: can you escape the * on line 55?

**Review by mairaw — APPROVED** [review]
LGTM @gewarren. Left a few comments for you to consider. Probably best to separate ML.NET content in future PRs of these kind?

File: docs/core/porting/windows-compat-pack.md (hunk 1 above)
Comment (gewarren) [reply docs/core/porting/windows-compat-pack.md]: It was about the missing period but doing this made it not show up as a false positive in my search query.

File: docs/csharp/tutorials/console-webapiclient.md (hunk 3 above)
Comment (gewarren) [reply docs/csharp/tutorials/console-webapiclient.md]: I reverted the change to this file.

**Pull request merged**